herddb-core/src/main/java/herddb/log/LogEntry.java

```diff
@@ -73,45 +86,45 @@ public int serialize(ExtendedDataOutputStream doo) throws IOException {
         doo.writeLong(timestamp); // 8
         doo.writeShort(type);  // 2
         doo.writeLong(transactionId); // 8
-        doo.writeUTF(tableSpace); // var
+        doo.writeUTF(""); // keep compatibility with v 0.2
```

Comment:
If an empty UTF string has already been evaluated in 38-51 why not writing directly its byte array without engoding it another time?